torch/csrc/jit/python/pybind_utils.h

```diff
@@ -1235,20 +1244,19 @@ inline py::object invokeScriptMethodFromPython(
       });
 }
 
-inline py::object invokeOperatorFromPython(
+inline std::pair<std::shared_ptr<Operator>, Stack> getOpWithStack(
```

Comment:
I am ok with changing the return type to bundle with Stack, but the fact that we can introducing a flag purely for ignoring a error message make me feels we are doing it in a wrong way.  maybe we should make the c10 op special handling (early return) inside this function directly?

Reply:
I agree with the sentiment that introducing a flag here to control whether we throw or not is not ideal. Although, I am a bit reluctant to add the c10 op special-casing here, in more general JIT code that is used elsewhere besides RPC.

The order of operators seems to be somewhat brittle and differs in different build modes (see internal discussion), which is why that special casing exists in the first place. Given that, until we can remove it completely I'd like to keep it only in RPC codebase to avoid this logic changing the JIT too much. Maybe we can remove the flag from here, and then special case the RPC code to retry (kind of similar to what we do currently in this diff).